skglm/datafits/group_task.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
```

Comment:
the file should be named `group.py` not `group_task.py` as there are no tasks here